demo-shell-ng2/package.json

```diff
@@ -6,18 +6,20 @@
   "scripts": {
     "ng": "ng",
     "start": "npm run server-versions && rimraf dist && ng serve --host 0.0.0.0 --app=0 --open --aot=true",
-    "start:dev": "npm run clean-ng2-component-angular && npm run server-versions && rimraf dist && ng serve --host 0.0.0.0 --app=1 pp-dev --proxy-config proxy.conf.js --open",
+    "start:dev": "npm run style:dev & npm run clean-ng2-component-angular && npm run server-versions && rimraf dist && ng serve --host 0.0.0.0 --app=1 pp-dev --proxy-config proxy.conf.js --open",
     "start:dist": "npm run server-versions && rimraf dist && ng serve --host 0.0.0.0 --disable-host-check --aot=false --prod --app=0",
     "build": "npm run server-versions && rimraf dist && ng build  -app=0",
-    "build:dev": "npm run server-versions && rimraf dist && ng build -app=1",
+    "build:dev": "npm run style:dev & npm run server-versions && rimraf dist && ng build -app=1",
+    "style:dev": "npm run webpack -- --config  config/webpack.style.js --progress --profile --bail --watch",
```

Comment:
Question:
What about production builds?

Reply:
the production build use the style already builded in the node_modules folder you don't need to rebuild it